### ISSUE TITLE
Deck Page Won't Load

### ISSUE THREAD

**Issue by Electrick-Warlok** [Razviar/mtgap#353]
Link: https://marvelsnap.pro/decks/deck-19-835z4/

Message Title: Error Occured!
Message Content: Ouch! Something went wrong and we were unable to render this page properly. First of all, please try reload this page with Ctrl+F5, clear browser cache manually, disable all adblocking software and try to reload the page. If the error still persists, please post an issue [on our GitHub](https://github.com/Razviar/mtgap/issues) to expedite solution for this problem.

Did all that. Still getting the error.

**Comment by Electrick-Warlok** [comment]
Happening again with https://marvelsnap.pro/decks/deck-17-2bzz9

**Comment by Razviar** [comment]
Thanks for reporting, we are investigating

**Comment by Razviar** [comment]
I'm unable to reproduce the issue, the page is loading fine for me, even when I turn on emulator of your account enviroment. 
So it means it's something on your side: adblock, cache, browser issue etc.

**Comment by Electrick-Warlok** [comment]
Yup, it's working for me now, too. Is it possible that newly submitted
decks needs time to propagate before being viewable? It seems like this
only happens to me with decks that are only a few hours old.

**Comment by rcasaleiro** [comment]
This is still happening, started playing 3 days ago and my decks still dont show up.

**Comment by rcasaleiro** [comment]
Well I wiped everything including the OVL location and it started working. I suspect I had old leftover data from a couple years ago.

**Comment by rcasaleiro** [comment]
Nvm, this happened again and it stopped updating.
